packages/service/package.json

```diff
@@ -9,9 +9,11 @@
     "@types/jest": "^26.0.24",
     "babel-jest": "^27.0.6",
     "jest": "^27.0.6",
-    "nodemon": "^2.0.9"
+    "nodemon": "^2.0.9",
+    "prisma": "^2.27.0"
```

Comment:
prisma should be a regular dependency I think

Reply:
Their docs say to run `yarn add prisma --dev` - I believe it's because prisma indicated here is just a cli tool

Reply:
oh, interesting. 🆒